src/Atmos/Model/AtmosModel.jl

```diff
@@ -506,7 +506,7 @@ equations.
 
     flux_pad = SVector(1, 1, 1)
     tend = Flux{FirstOrder}()
-    _args = (state = state, aux = aux, t = t, direction = direction)
+    _args = (; state, aux, t, direction)
```

Comment:
I think this syntax was added 1.5. Our compat bound is still 1.4. Should we update it ?

Reply:
That's a good point. I think we're using this syntax elsewhere, but the compat probably should match.

Reply:
Opened #1955 to do this separately.